scripts/upgrade

```diff
@@ -20,6 +20,7 @@ domain=$(ynh_app_setting_get $app domain)
 path_url=$(ynh_app_setting_get $app path)
 is_public=$(ynh_app_setting_get $app is_public)
 final_path=$(ynh_app_setting_get $app final_path)
+phpversion=$(ynh_app_setting_get $app phpversion)
```

Comment:
It looks like there's no handling of "this setting doesnt exists yet" which may make everything explode when later we use it ?